admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```
@@ -109,6 +109,105 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
```

```suggestion
    gap: 1.5625rem;
```

I think if we use `rem` on margin and paddings, no reason to not use it on gaps, as we wants it to adapt on font-size change

---

You're right I copied this from the mockup directly I converted the other ones but I forgot this one, my bad